soc/arm/nordic_nrf/nrf52/Kconfig.soc

```diff
@@ -361,3 +361,21 @@ config NRF_ENABLE_ICACHE
 	bool "Enable the instruction cache (I-Cache)"
 	depends on SOC_NRF52832 || SOC_NRF52833 || SOC_NRF52840
 	default y
+
+config NRF52_ANOMALY_132_DELAY_US
+	int "Anomaly 132 workaround delay (microseconds)"
+	default 220
+	depends on NRF52_ANOMALY_132_WORKAROUND
+	help
+	  Due to Anomaly 132 LF RC source may not start if restarted in certain
+	  window after stopping (230 us to 330 us). Software reset also stops the
+	  clock so if clock is initiated in certain window, it may also fail to
+	  start at reboot. A delay is added before starting LF clock to ensure that
+	  anomaly conditions are not met. Delay should be long enough to ensure that
+	  clock is started later than 330 us after reset. If crystal oscillator(XO)
```

Comment:
If this is the case, could we just make the default 330us, instead? Then the solution is deterministic.

Reply:
i agree, if someone starts tweaking startup time it may have it shorter than empty app and if he is tweaking then sooner or later he will find out this delay and tune it.